tutorials/feature-use/Migration_from_Prophet.ipynb

```diff
@@ -0,0 +1,293 @@
+   "metadata": {},
+   "source": [
+    "### Trend\n",
+    "\n",
+    "In both frameworks, the trend can be configured during the init of the forecaster object. Pay attention to that the naming of the attributes might be slightly different between the two (eg. `changepoint_range` vs. `changepoints_range`)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Prophet\n",
+    "p = Prophet(growth=\"linear\", n_changepoints=10, changepoint_range=0.5)\n",
+    "p.fit(df)\n",
+    "\n",
+    "# NeuralProphet\n",
+    "np = NeuralProphet(growth=\"linear\", n_changepoints=10, changepoints_range=0.5)\n",
+    "np.fit(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Seasonality\n",
+    "\n",
+    "In both frameworks, custom seasonality can be added using the `add_seasonality(...)` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Prophet\n",
+    "p = Prophet(weekly_seasonality=False)\n",
+    "p = p.add_seasonality(name=\"monthly\", period=30.5, fourier_order=5)\n",
+    "p.fit(df)\n",
+    "\n",
+    "# NeuralProphet\n",
+    "np = NeuralProphet(weekly_seasonality=False)\n",
+    "np = np.add_seasonality(name=\"monthly\", period=30.5, fourier_order=5)\n",
+    "np.fit(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Country holidays\n",
+    "\n",
+    "The `add_country_holidays(...)` function works identical in both framework."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Prophet\n",
+    "p = Prophet()\n",
+    "p = p.add_country_holidays(country_name='US')\n",
+    "p.fit(df)\n",
+    "\n",
+    "# NeuralProphet\n",
+    "np = NeuralProphet()\n",
+    "np = np.add_country_holidays(country_name='US')\n",
+    "np.fit(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Future regressors"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def nfl_sunday(ds):\n",
+    "    date = pd.to_datetime(ds)\n",
+    "    if date.weekday() == 6 and (date.month > 8 or date.month < 2):\n",
+    "        return 1\n",
+    "    else:\n",
+    "        return 0\n",
+    "df_nfl = df.copy()\n",
+    "df_nfl['nfl_sunday'] = df_nfl['ds'].apply(nfl_sunday)\n",
+    "\n",
+    "# Prophet\n",
+    "p = Prophet()\n",
+    "p = p.add_regressor('nfl_sunday')\n",
+    "p.fit(df_nfl)\n",
+    "future_p = p.make_future_dataframe(periods=30)\n",
+    "future_p['nfl_sunday'] = future_p['ds'].apply(nfl_sunday)\n",
+    "_ = p.predict(future_p)\n",
+    "\n",
+    "# NeuralProphet\n",
+    "np = NeuralProphet()\n",
+    "future_np = np.make_future_dataframe(df_nfl, periods=30)\n",
```

Comment:
align position with prophet

Reply:
This does actually not work since NeuralProphet expects an `regressors_df` to be passed to the `make_future_dataframe()` function, which would require significantly more lines of code. Calling `make_future_dataframe()` before the fit function acts as a workaround for that.

Reply:
oh, ok, thanks